buildenv/jenkins/jobs/builds/Build-Test-Any-Platform

```diff
@@ -20,6 +20,16 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
 
+// Discard old builds
+NUM_ARTIFACTS = 5
+NUM_BUILDS = 10
+DAYS_BUILDS = 60
+if (params.ghprbPullId) {
+    // Keep last 50 of each PullRequest build
+    NUM_BUILDS = 50
+}
+properties([buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: NUM_ARTIFACTS, daysToKeepStr: DAYS_BUILDS, numToKeepStr: NUM_BUILDS))])
```

Comment:
If you move this code into a fuction in variables-functions then you avoid duplicating the code.